test/pylib/scylla_cluster.py

```diff
@@ -703,26 +709,27 @@ async def start(self) -> None:
         await self.site.start()
         self.is_running = True
 
-    async def _before_test(self, test_name: str) -> None:
+    async def _before_test(self, test_case_name: str) -> None:
         if self.cluster.is_dirty:
             await self.clusters.steal()
             await self.cluster.stop()
             await self._get_cluster()
-        logging.info("Leasing Scylla cluster %s for test %s", self.cluster, test_name)
-        self.cluster.before_test(self.test_name)
+        self.current_test_case_full_name = f'{self.test_uname}::{test_case_name}'
+        logging.info("Leasing Scylla cluster %s for test %s", self.cluster, self.current_test_case_full_name)
```

Comment:
I think it's rather obvious it's the current test case full name, so the word 'current_' could be omitted. Entirely up to you.